vcr/pe/presentation_submission.go

```diff
@@ -149,13 +149,16 @@ func (b *PresentationSubmissionBuilder) Build(format string) (PresentationSubmis
 
 	// the verifiableCredential property in Verifiable Presentations can be a single VC or an array of VCs when represented in JSON.
 	// go-did always marshals a single VC as a single VC for JSON-LD VPs. So we might need to fix the mapping paths.
-	if format == vc.JSONLDPresentationProofFormat {
-		for _, signInstruction := range nonEmptySignInstructions {
-			if len(signInstruction.Mappings) == 1 {
-				signInstruction.Mappings[0].Path = "$.verifiableCredential"
-			}
+
+	// todo the check below actually depends on the format of the credential and not the format of the VP
```

Comment:
I think this isn't right?

Reply:
It did create an error when running the e2e-test. But "fixing" it now is a bit pro-active since VC formats are going to be changed anyway.